config/examples/redhat_customer_portal.yml

```diff
@@ -1,19 +1,15 @@
 base: &default
-  enabled: true
-  client_id:
-  client_secret:
-  realm:
-  flow:
-  entitlements_notifications_enabled: true
+  enabled: <%= ENV.fetch('RH_CUSTOMER_PORTAL_ENABLED', false) %>
+  client_id: <%= ENV.fetch('RH_CUSTOMER_PORTAL_CLIENT_ID', '3scale') %>
+  client_secret: <%= ENV['RH_CUSTOMER_PORTAL_CLIENT_SECRET'] %>
```

Comment:
maybe escape with `to_json` here as well

Reply:
Why do we need it?

Reply:
I don't know what characters can be used. If it is a user password, it can contain many characters. If it is auto-generated, then could be alright.